source/includes/api/_subscriptions.md

```diff
@@ -1,704 +1,811 @@
-datasetsQuery |              Subscriptions to subscribable datasets               |              Array | Yes (unless `datasets` is specified)
--- id         |                           Id of dataset                           |           ObjectId | Yes (unless `datasets` is specified)
--- type       | Type of subscription defined in the dataset                       |               Text | Yes (unless `datasets` is specified)
--- params     | Geographic area of the subscription                               |             Object | Yes (unless `datasets` is specified)
-env           |  Environment of the subscription. Set to `production` by default  |             String | No
+```shell
+curl -X GET "https://api.resourcewatch.org/v1/subscriptions/<subscription_id>/confirm"
+```
 
-## Unsubscribe
+Upon creation of the subscription, a confirmation email is sent to the email of the user who created the subscription. This email will contain a link, which you will need to click in order to confirm the subscription. You can also confirm the subscription by performing a GET request to the `v1/subscriptions/:id/confirm` endpoint, as exemplified on the side.
 
-To unsubscribe from a subscription:
+*Manually calling this endpoint will redirect you to the GFW application.*
 
-```shell
-curl -X GET https://api.resourcewatch.org/v1/subscriptions/:id/unsubscribe \
--H "Authorization: Bearer <your-token>"
-```
+### Resending the subscription confirmation email
 
-## Delete subscription
+> Example GET request to resend the confirmation email for a subscription:
 
 ```shell
-curl -X DELETE https://api.resourcewatch.org/v1/subscriptions/:id/unsubscribe \
+curl -X GET "https://api.resourcewatch.org/v1/subscriptions/<subscription_id>/send_confirmation" \
 -H "Authorization: Bearer <your-token>"
 ```
 
-To delete a subscription, use the following DELETE endpoint.
+For convenience, the RW API offers an additional endpoint to resend the confirmation email. In order to do this, you should perform a GET request to the `v1/subscriptions/:id/send_confirmation`. As with most of the other subscription endpoints, please keep in mind that you must be authenticated in order to use this endpoint.
 
-If the request comes from another microservice, then it is possible to delete subscriptions belonging to other users. Otherwise, you can only delete subscriptions if you are the owner of the subscription.
+*Manually calling this endpoint will redirect you to the GFW application.*
 
+### Unsubscribing
 
+> Example GET request to unsubscribe:
 
-## Subscription statistics
-
-Statistics endpoints require authentication by an ADMIN user.
+```shell
+curl -X GET "https://api.resourcewatch.org/v1/subscriptions/<subscription_id>/unsubscribe" \
+-H "Authorization: Bearer <your-token>"
+```
 
-### General subscription statistics
+**Note: unsubscribing is equivalent to deleting the subscription**.
```

Comment:
Just functionally, you mean, or does it actually delete the subscription?

Reply:
It actually fully deletes the subscription - should we try to make it clearer?

Reply:
Would just make it a little more direct - just say, "When you unsubscribe, the underlying subscription resource is deleted."

Reply:
done :+1: